Add a note about client schemas

mike-marcacci opened the pull request:
I updated the CLI docs and apollo readme to note that they are now
specified in the --includes glob.

Fixes #786.

trevor-scheer commented:
Hey Mike, thanks for adding this. The README is updated automatically when we publish the package, so if you don't mind reverting the changes to that file, this should be all set.

Looks like this might need a `prettier` run as well, though I think that should be in a precommit hook?

mike-marcacci commented:
Hi @trevor-scheer! I must have missed your comment. Fixed and fixed; prettier wasn't run because I never actually ran `npm install` (since it was a docs-only change) and so git hooks weren't set up.

trevor-scheer commented:
Thanks @mike-marcacci! 🏆
